config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml

```diff
@@ -27,9 +27,8 @@ periodics:
         - -xc
         - >
           make WHAT="github.com/onsi/ginkgo/v2/ginkgo k8s.io/kubernetes/test/e2e/e2e.test" &&
-          curl -sSL --output "${PATH%%:*}/kind" https://github.com/kubernetes-sigs/kind/releases/download/v0.19.0/kind-linux-amd64 &&
-          chmod u+x "${PATH%%:*}/kind" &&
-          kind build node-image --base-image gcr.io/k8s-staging-kind/base:v20230515-01914134-containerd_v1.7.1@sha256:468fc430a6848884b786c5cd2f1c03e7a0977f04fb129a2cda2a19ec986ddacb --image dra/node:latest &&
```

Comment:
if this gets re-reverted for some reason: the previous version was missing a `.` or `$(pwd)` or otherwise passing the path to k8s sources and for some reason the k8s source dir auto-detection fallback isn't working